client/state/sites/plans/selectors.js

```diff
@@ -98,6 +118,32 @@ export function getPlanDiscountedRawPrice(
 }
 
 /**
+ * Returns a plan price before discount
+ *
+ * @param  {Object}  state         global state
+ * @param  {Number}  siteId        the site id
+ * @param  {String}  productSlug   the plan product slug
+ * @param  {Boolean} isMonthly     if true, returns monthly price
+ * @return {Number}                plan raw price
+ */
+export function getPlanRawPrice(
```

Comment:
It's a good habit in helping distinguish the selectors, especially if there's a chance of conflicting selector names, and therefore most future proof in the case that something like global selectors does come to light (`import { ... } from 'state/selectors';` being the goal, whether or not that takes the form of #5954).